daemon/src/maker_cfd.rs

```
@@ -579,7 +579,7 @@ where
                 taker_id,
                 command: TakerCommand::NotifyOrderAccepted { id: order_id },
             })
-            .await?;
+            .await??;
```

This is a behavioural change, can we easily use `spawn_fallible` here?

---

Or, in case you want to keep it, can we have it as a separate commit just in case we need to bisect a bug?

---

Is it not as a separate commit already? Can't really split it further, without introducing and removing the spawn fallible in a commit it-between. 

IMHO we really should stop the contract setup if it fails instead of just logging it.

---

yeah, the behaviour should result in the same thing, in the end we always log an error (just on the inside now instead of the outside) if I understand it correctly.

---

I see two commits:

1. Removing log_error and use spawn_fallible on the calling side

2. Use xtra_productivity to remove boilerplate.

I would like to retain functionality for now, feel free to propose a follow-up where we discuss changing the behaviour.

---

I don't think it would actually be a behavioral change, as it will still result in a `log_error()` in the maker_cfd actor handler, it will just stop stop going further after this failure, as it is currently. I can be even more explicit about this behavioral change in the commit perhaps?

---

ok, I'll create a follow-up with that simplification separately.

---

Started #418